Добавляет рецепт «Как добавить пользователей в репозиторий» (#4722)
doka-guide/content: recipes/github-add-user-to-repo/index.md

---
title: "Как добавить пользователей в репозиторий на GitHub"
description: "Работайте на GitHub совместно с другими пользователями."
authors:
  - igsekor
related:
  - tools/version-control
  - tools/git-cli
  - tools/github-actions
tags:
  - article
---

## Задача

> Добавить пользователей в репозиторий для дальнейшей совместной работы

Reviewer: ```suggestion
Добавить пользователей в репозиторий для дальнейшей совместной работы.
```

Тчк :)

## Готовое решение

> Необходимо перейти в настройки репозитория, которые находятся на последней вкладке «Settings»:
> 
> ![Вкладки в верхней части окна для работы с репозирием. Выбрана вкладка с настройками «Settings»](images/repository-tabs.png)

Reviewer 2: ```suggestion
Перейдите в репозиторий и найдите панель вкладок в верхней части окна для работы с репозиторием. В нём выберете последнюю вкладку «Settings».

![Вкладки для работы с репозирием](images/repository-tabs.png)
```

> Далее необходимо выбрать раздел слева, посвящённый взаимодействию пользователей в внутри репозитория:

Reviewer 2: ```suggestion
Потом нужно выбрать раздел «Collaborators» в подразделе «Access» из бокового меню слева. Он посвящён взаимодействию пользователей внутри репозитория.
```

> ![Раздел настроек репозитория «Collabarators», в котором можно управлять пользователями](images/repository-collaborators.png)

Reviewer 2: ```suggestion
![Пункт «Collabarators» в настройках репозитория](images/repository-collaborators.png)
```

> Добавить нового участника можно с помощью кнопки «Add people», после нажатия которой откроется всплывающее окно с возможностью выбора имени пользователя (просто начните набирать и выберете пользователя из погрузившегося списка):

Reviewer: ```suggestion
Добавить нового участника можно с помощью кнопки «Add people», после нажатия которой откроется всплывающее окно с возможностью выбора имени пользователя (просто начните набирать и выберите пользователя из подгрузившегося списка):
```

Reviewer 2: И я бы двоеточие предложила убрать в конце последнего предложения и заменить на точку.

![Окно добавления пользователя в репозиторий](images/github-add-collaborator.png)

> После нажатия на нужном пользователе, название кнопки поменяется на «Add \<имя пользователя\> to this repository», и она станет активной. Останется нажать на эту кнопку, чтобы отослать приглашение пользователю, которое придёт пользователю на электронную почту. После подтверждения, пользователь получит доступ к репозиторию.

Reviewer 2: ```suggestion
После нажатия на имя нужного пользователя, название кнопки поменяется на «Add \<имя пользователя\> to this repository», и она станет активной. Останется нажать на эту кнопку, чтобы отослать приглашение пользователю. Оно придёт ему на электронную почту. Пользователь получит доступ к репозиторию после подтверждения.
```
